src/pycolmap/estimators/similarity_transform.cc

```diff
@@ -0,0 +1,66 @@
+#include "colmap/estimators/similarity_transform.h"
+
+#include "colmap/math/random.h"
+#include "colmap/optim/loransac.h"
+#include "colmap/util/logging.h"
+
+#include "pycolmap/pybind11_extension.h"
+#include "pycolmap/utils.h"
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+using namespace colmap;
+using namespace pybind11::literals;
+namespace py = pybind11;
+
+py::typing::Optional<py::dict> PyEstimateSim3dRobust(
```

Comment:
Equivalent to the existing EstimateSim3d function, we could create a EstimateSim3dRobust function on the C++ side instead of the binding side. I would recommend keeping the binding layer as minimal as possible.

Reply:
Done, PTAL.